### PR TITLE
NEWS.md: add release notes for v0.12.0.

### PR DESCRIPTION
**Pull request by jameshcorbett**
Problem: there are no release notes for v0.12.0.

Add them.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+flux-coral2 0.12.0 - 2024-04-10
+-------------------------------
+
+### New Features
+
+ * dws: add support for draining Offline nodes (#140)
+ * dws: k8s special exitcode (#146)
+ * dws: flux finalizer (#141)
```
Comment (grondo) [inline NEWS.md:8]: I suggest rewording this entry since it isn't clear what's changed. (On the off chance someone other than us looks at the release notes). However, up to you if you don't feel it's important.

**Review by grondo — APPROVED** [review]
LGTM, just one minor suggestion inline.

File: NEWS.md (hunk 1 above)
Comment (jameshcorbett) [reply NEWS.md]: Good catch, thanks! Fixed.

**Comment by jameshcorbett** [comment]
Force-pushed to add an entry for #149 and address your comment.

**Comment by grondo** [comment]
Already approved so feel free to set MWP when ready
